Flow.Launcher/Converters/QuerySuggestionBoxConverter.cs

```diff
@@ -12,31 +12,20 @@ public class QuerySuggestionBoxConverter : IMultiValueConverter
     {
         public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
         {
-            if (values.Length != 3)
-            {
-                return string.Empty;
-            }
-            var QueryTextBox = values[0] as TextBox;
-
-            var queryText = (string)values[2];
-
-            if (string.IsNullOrEmpty(queryText))
+            if (
+                values.Length != 3 ||
+                values[0] is not TextBox queryTextBox ||
+                values[1] is null ||
+                values[2] is not string queryText ||
+                string.IsNullOrEmpty(queryText)
+            )
                 return string.Empty;
-
-            // second prop is the current selected item result
-            var val = values[1];
-            if (val == null)
-            {
-                return string.Empty;
-            }
-            if (!(val is ResultViewModel))
-            {
-                return System.Windows.Data.Binding.DoNothing;
-            }
+            
+            if (values[1] is not ResultViewModel selectedItem)
+                return Binding.DoNothing;
```

Comment:
Separate checks instead of one list pattern because I need to return `Binding.DoNothing` in one specific case.